core/services/ocrcommon/transmitter.go

```diff
@@ -203,3 +217,55 @@ func (t *ocr2FeedsTransmitter) forwarderAddress(ctx context.Context, eoa, ocr2Ag
 
 	return forwarderAddress, nil
 }
+
+type ocr2FeedsDualTransmission struct {
+	transmitter ocr2FeedsTransmitter
+
+	secondaryContractAddress common.Address
+	secondaryFromAddress     common.Address
+	secondaryMeta            map[string][]string
+}
+
+func (t *ocr2FeedsDualTransmission) CreateEthTransaction(ctx context.Context, toAddress common.Address, payload []byte, txMeta *txmgr.TxMeta) error {
+	// Primary transmission
+	errPrimary := t.transmitter.CreateEthTransaction(ctx, toAddress, payload, txMeta)
+	errPrimary = errors.Wrap(errPrimary, "skipped primary transmission")
```

Comment:
nit: I think we no longer want to use `"github.com/pkg/errors"` and we only want to use the std lib, so that could change into: 
```suggestion
	errPrimary = fmt.Errorf("skipped primary transmission: %w", errPrimary)
```

Reply:
Changed in [`4985cf9` (#15202)](https://github.com/smartcontractkit/chainlink/pull/15202/commits/4985cf950f33f93c43cbe1c01ee1dde60bc87a72)